src/main/java/io/confluent/connect/elasticsearch/bulk/BulkProcessor.java

```diff
@@ -125,15 +132,17 @@ private Runnable farmerTask() {
     return new Runnable() {
       @Override
       public void run() {
-        log.debug("Starting farmer task");
-        try {
-          while (!stopRequested) {
-            submitBatchWhenReady();
+        try (LogContext context = logContext.create("Farmer1")) {
```

Comment:
What's the significance of the `1` here, if it's just a constant number it seems like noise.
```suggestion
        try (LogContext context = logContext.create("Farmer")) {
```

Reply:
I left this as `Farmer1` because it aligns the log messages cleanly with `. I'd prefer to keep it unless you *really* prefer not to.

Reply:
haha okay.
Maybe a space instead?
```suggestion
        try (LogContext context = logContext.create("Farmer ")) {
```

Reply:
Oh okay, I didn't realize thread had a number after it as well.
This is fine as-is I suppose.